docs/en/latest/getting-started.md

```diff
@@ -21,7 +21,7 @@ title: Getting Started
 #
 -->
 
-## Getting Started
+## Getting Started {#apisix}
```

Comment:
This breaks the render in the GitHub markdown preview (maybe also the cross reference). Can we add the mark into '<!-- -->' and take it away from the title?

Reply:
No we can't do that, and i have tested this on my system it works fine

Reply:
its not the title , the title is above the license text

Reply:
@iamayushdas I suggest submitting PRs `on one repo` for test & preview, let's have a preview here https://github.com/apache/apisix-dashboard/pull/1951 first.

Reply:
okay

Reply:
It is **the title** from the view of the markdown version. See https://github.com/apache/apisix/blob/master/docs/en/latest/getting-started.md

Reply:
I can't accept such a scar on the title. The solution is quite a hack. How can a mark be put into the content, especially the content is obvious to everyone?

Reply:
@spacewander Sir, first of all I am extremely sorry flooding PRs without discussing,
My PROPOSAL:
Docusaurus has opted this way for adding id's to be more precise. they have changed the md style in such a way that we can add precise id without harming our website
reference md file - https://github.com/facebook/docusaurus/blob/master/website/docs/api/plugins/plugin-content-blog.md
screenshot: 
<img width="1323" alt="image" src="https://user-images.githubusercontent.com/40708551/123247795-df6d4d80-d504-11eb-9c1f-dbef20dc2225.png">


website created from above md file: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog
screenshot: 
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog
<img width="1320" alt="Screenshot 2021-06-24 at 3 55 22 PM" src="https://user-images.githubusercontent.com/40708551/123247579-aaf99180-d504-11eb-8b1f-ab05463c93a1.png">

now please visit this issue: https://github.com/apache/apisix-website/issues/363